src/usr/bin/google_set_multiqueue

```diff
@@ -269,7 +311,12 @@ fi
 # enp140s0
 
 # IRQ binding for numa 0, CPUs [0, 51] and [104, 155] are for numa 0.
-numa0_irq_start=1
+irq_ranges=()
+get_vcpu_ranges irq_ranges
+echo "Binding vCPUs on NUMA0 [${irq_ranges[0]} ${irq_ranges[1]}], [${irq_ranges[2]} ${irq_ranges[3]}]}"
+echo "Binding vCPUs on NUMA1 [${irq_ranges[4]} ${irq_ranges[5]}], [${irq_ranges[6]} ${irq_ranges[7]}]}"
+
+numa0_irq_start=${irq_ranges[0]}
 find /sys/class/net -type l | xargs -L 1 realpath | grep '/sys/devices/pci' | sort | xargs -L 1 basename | while read nic_name; do
   # For non-gvnic devices (e.g. mlx5), the IRQ bindings will be handled by the device's driver.
   if ! is_gvnic "$nic_name"; then
```

Comment:
Pre-existing code here, but.. Public A3 Documentation doesn't say that gvnic is required. Does this need to apply to virtio NICs as well?

Reply:
All accelerator platforms of gen3 and after (A3/A3-mega) only offers Gvnic, virtionet is not supported, while A3-ultra supports both Gvnic and MLX5, so we'd want to skip IRQ binding on non-gvnic devices.